.eslintrc

```diff
@@ -5,12 +5,12 @@
     "prettier"
   ],
   "rules": {
-    "no-async-promise-executor": 0,
-    "no-constant-condition": 0,
-    "@typescript-eslint/no-empty-function": 0,
-    "@typescript-eslint/no-unused-vars": 0,
-    "@typescript-eslint/no-explicit-any": 0,
-    "@typescript-eslint/no-non-null-assertion": 0
+    "no-async-promise-executor": 1,
+    "no-constant-condition": 1,
+    "@typescript-eslint/no-empty-function": 1,
+    "@typescript-eslint/no-unused-vars": 1,
+    "@typescript-eslint/no-explicit-any": 1,
+    "@typescript-eslint/no-non-null-assertion": 1,
```

Comment:
These are the things adding the warnings after we get rid of the warnings we can then set this to `2` so they'll start throwing errors

We can also add or remove these options if we'd like